### ISSUE TITLE
Python error

### ISSUE THREAD

**Issue by DaZeNuk3r** [persianhydra/Xeexe-TopAntivirusEvasion#23]
![ubuntu1804_GDNnZM3thz](https://user-images.githubusercontent.com/63490802/128699273-94a62f09-f041-474f-92bb-1cf1fc92070d.png)
![ubuntu1804_KaB0qrC4LX](https://user-images.githubusercontent.com/63490802/128699322-065f0834-52ee-4a27-8f18-3cee9e3a4609.png)

**Comment by CYBERDEVILZ** [comment]
Clearly as the error says, its an issue of indentation. You've used spaces and tabs to indent your code. In a Python program, you should stick to using either one of these two methods of indentation.

**Comment by DaZeNuk3r** [comment]
this code is unedited.

**Comment by CYBERDEVILZ** [comment]
Yeah I get it. But its an easy fix. You can manually add the  indents yourself. Have you tried doing that?
